tests/cpp/dlr_allocator_test.cc

```diff
@@ -25,6 +24,10 @@ void test_free(void* ptr) { throw dmlc::Error("Using custom free"); }
 
 void* test_memalign(size_t alignment, size_t size) { throw dmlc::Error("Using custom memalign"); }
 
+DLRMallocFunctionPtr test_malloc_fn = test_malloc;
```

Comment:
Could you make these variables members of CustomAllocatorTest?

Reply:
fixed